src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -26,7 +26,7 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
         const { initializationData } = getPaymentMethodOrThrow(methodId, gatewayId);
         const { orderId, buttonStyle } = initializationData;
 
-        if (orderId) {
+        if (initializationData && orderId) {
```

Comment:
Have you tested this? It doesn't seem right because `initializationData` is already destructed in the line above. So if  `initializationData` is `null`, it'll break in the line above rather than this one. I think you should do something like this instead:

```ts
const { orderId, buttonStyle } = initializationData ?? {};
```

Reply:
fixed